metau-capstone/Info.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>sheets_api_key</key>
```

Comment:
nit: to make these variable clear, name it to `google_sheets_api_key`

Reply:
done!